code/modules/clothing/glasses/engine_goggles.dm

```diff
@@ -19,7 +19,7 @@
 
 	var/list/modes = list(MODE_NONE = MODE_MESON, MODE_MESON = MODE_TRAY, MODE_TRAY = MODE_RAD, MODE_RAD = MODE_NONE)
 	var/mode = MODE_NONE
-	var/range = 1
+	var/range = 6
```

Comment:
Congratulations, this change officially makes tray scanning googles absolutely useless.
Perhaps you may want to push that 6 there and keep this a 4, no?